modules/social_features/social_event/config/install/field.field.node.event.field_event_send_confirmation.yml

```diff
@@ -0,0 +1,25 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_event_send_confirmation
+    - node.type.event
+  enforced:
+    module:
+      - social_event
+id: node.event.field_event_send_confirmation
+field_name: field_event_send_confirmation
+entity_type: node
+bundle: event
+label: 'Send confirmation after standalone enrolment'
```

Comment:
Does this label correct?

Reply:
@rollki I added readable label for the field more-less described what it does. But in the form users will see description of the field (see [here](https://github.com/goalgorilla/open_social/pull/2853/files#diff-4c305b27578c9c82905e13f0df9785fa52e1ec815883ca4fabf05b624fda0f8cR745))